components/css_selector.rst

```diff
@@ -47,11 +47,14 @@ The CssSelector Component
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The component's only goal is to convert CSS selectors to their XPath
-equivalents::
+equivalents, using :method:`Symfony\\Component\\CssSelector\\CssSelectorConverter::toXPath`
 
-    use Symfony\Component\CssSelector\CssSelector;
+.. code-block:: php
```

Comment:
I think you should still use the double colon at the end of the previous paragraph and omit the `code-block` directive here.